src/Verifier/SAW/Simulator/Value.hs

```diff
@@ -80,6 +82,8 @@ type family VBool l :: Type
 type family VWord l :: Type
 -- | Integers for value instantiation 'l'
 type family VInt  l :: Type
+-- | SMT arrays for value instantiation 'l'
+type family VArray l :: Type
```

Comment:
It looks like the neither the `Concrete`  nor `RME` backend implements this type family. Is this intentional? Or would it be better to explicitly instantiate `VArray Concrete` and `VArray RME` to at least some dummy type like `()` for now?

Reply:
👍